packages/plugin-table/src/Table.ts

```diff
@@ -139,8 +144,51 @@ export class Table<T extends JWPluginConfig> extends JWPlugin<T> {
         const cells = range.targetedNodes(TableCellNode);
         for (const cell of cells) {
             for (const managedCell of cell.managedCells) {
-                managedCell.managerCell = null;
+                managedCell.unmerge();
             }
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Return true if the given array of cells forms a rectangle in the table
+     * grid.
+     *
+     * @param cells
+     */
+    _isRectangle(cells: TableCellNode[]): boolean {
+        // Add managed cells to the list.
+        for (const cell of cells.slice()) {
+            if (cell.managedCells.size) {
+                cells.push(...cell.managedCells);
```

Comment:
Wait, you're modifying the array you were given. Rather than do `.slice` in the `for`, you can do `cells = [...cells]`

Reply:
Then I'd do `cells = [...cells]` above then `for (const cell of [...cells])` for the loop.

Reply:
... which is what I did :-)